lib/simplecov-console.rb

```diff
@@ -73,17 +75,28 @@ def format(result)
       return
     end
 
+    if SimpleCov::Formatter::Console.files_to_filter.empty?
+      files = result.files
+    else
+      files = result.files.select  do |file|
+        SimpleCov::Formatter::Console.files_to_filter.include?(file.filename)
```

Comment:
It looks like `files_to_filter` would be expected to include absolute paths to each file, is that what you meant for here? I think we can modify this to at least use a path relative to the current project, what do you think?